### PR TITLE
Add the LICENSES directory for install and use the corrected name in …

### PR DESCRIPTION
**Pull request by jensschmeink**
…the about dailog

## Checklist:

- [ ] A test for the new functionality was added (if applicable).
- [x] All tests run without failure.
- [ ] The changelog has been extended, if this MR contains important changes from the users's point of view.
- [x] The new code complies with the GTlab's style guide.
- [ ] New interface methods / functions are exported via `EXPORT`. Non-interface functions are NOT exported.
- [x] The number of code quality warnings is not increasing.

/label ~InProgress

### CODE REVIEW

**Review by rainman110 — CHANGES_REQUESTED** [review]
As much as I'd like to approve, I don't think that installing unconditionally to /LICENSES is a good idea.

This is a non-standard director, in particular on Linux. Assume, you install GTlab to /usr, then all GTlab licenses are installed to /usr/LICENSES. This is definitely not the right place. We already use non-standard directory layouts but we should not make it worse.

Instead, I propose to install to `share/gtlab/LICENSES` or a similar location.

**Comment by rainman110** [comment]
Oh no! Please don't differentiate between linux and windows. Also never install to a absolute location (/usr).

Just install to CMAKE_INSTALL_PREFIX/share/gtlab/licenses, i.e. 

```
install(DIRECTORY LICENSES/ share/gtlab/licenses)
```

You just have done so much work 😞 . I am sorry

File: src/app/dialogs/gt_aboutdialog.cpp [hunk 1]
```diff
@@ -371,30 +371,24 @@ GtAboutDialog::openModuleDetails(const QModelIndex& indexOfModule) const
...
-    QDir appLicenseDir(appDir.absolutePath() + QDir::separator() + "Licenses");
+    QString licenseDirPath = gtApp->licenseFolder();
 
-    if (appLicenseDir.exists())
+    if (licenseDirPath.isEmpty())
     {
-        QDesktopServices::openUrl(appLicenseDir.absolutePath());
+        gtError() << tr("No licenses information could be found.");
+        gtError() << tr("Your installation might be broken.");
+        gtError() << tr("Please contact the developers for further information.");
```
Comment (jensschmeink) [inline src/app/dialogs/gt_aboutdialog.cpp:380]:
```suggestion
```

**Review by rainman110 — APPROVED** [review]
Nice work :)

**Pull request merged**